README.md

```diff
@@ -3,48 +3,73 @@
 
-## Installation
+## Getting Started
 
-You can install structurizr-site-generatr using Homebrew:
+To get started with the Structurizr Site Generatr, you can either:
+
+- Install it to your local machine (requires Homebrew), or
+- Execute it on your local machine via a container (requires Docker)
+
+### Installation
+
+As this approach relies on [Homebrew](https://brew.sh/), ensure this is already installed. For Windows and other operating systems not supported by Homebrew, please use the [Docker approach](#docker) instead.
+
+To install Structurizr Site Generatr execute the following commands in your terminal:
 
 ```shell
 brew tap avisi-cloud/tools
 brew install structurizr-site-generatr
+
 structurizr-site-generatr --help
 ```
 
-## Docker image
+Periodically, you would have to update your local installation to take advantage of any new [Structurizr Site Generatr releases](https://github.com/avisi-cloud/structurizr-site-generatr/releases).
+
+### Docker
 
-You can also use the Docker image:
+Though local installation is recommended for development where possible, Structurizr Site Generatr is also a packaged as a Docker image. Therefore to use this approach, ensure [Docker](https://www.docker.com/) is already installed. Additionally, for Windows 10+ users, you may want to take advantage of [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) (Windows Subsystem for Linux). Both Docker and WSL2 are topics too vast to repeat here, so you are invited to study these as prerequisite learning for this approach.
+
+Then to download our packaged image, consider the [Structurizr Site Generatr releases](https://github.com/avisi-cloud/structurizr-site-generatr/releases) and choose the version you wish to use. Then, in your terminal, execute the following:
 
 ```shell
 docker pull ghcr.io/avisi-cloud/structurizr-site-generatr:<version>
-docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:<version> --help
 ```
 
-Replace `<version>` with the version you want to use.
+Note: The `<version>` suffix should be replaced with your chosen version number. For example `docker pull ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9`.
+
+<!-- FEATURE: By tagging the latest release with `latest` (see below), this step can be omitted as Docker automatically pulls any image unless cached locally -->
 
-### Verify image with cosign
+Once downloaded, you can execute Structurizr Site Generatr via a temporary Docker container by executing the following in your terminal:
 
+```shell
+docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:<version> --help
 ```
+
+Note: The `<version>` suffix should be replaced with same version number of your downloaded image. For example `docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9 --help`.
+
+<!-- FEATURE: By tagging the latest release with `latest`, the version can be omitted. By default Docker will pull the `latest` image, which will automatically pull on new releases -->
```

Comment:
Could you remove this comment and file an issue instead?

Reply:
Sure.. I'll file an issue by default when I find an enhacement to suggest (rather than leaving comments for you to assess first).